test/e2e/common_suite_test.go

```diff
@@ -157,8 +165,24 @@ func (tc *testCase) run() {
 			}
 
 			if tc.pod != nil {
-				if err := client.Resources(tc.pod.Namespace).List(ctx, &podlist); err != nil {
-					t.Fatal(err)
+
+				if tc.podWatcher {
+					if err := client.Resources("confidential-containers-system").List(ctx, &podlist); err != nil {
+						t.Fatal(err)
+					}
+					for _, caaPod := range podlist.Items {
+						if caaPod.Labels["app"] == "cloud-api-adaptor" {
+							imagePullTime, err := watchImagePullTime(ctx, client, caaPod, *tc.pod)
+							if err != nil {
+								t.Fatal(err)
+							}
+							t.Logf("Pod with larger image started successfully")
+							t.Logf("ImageName:%s", tc.pod.Spec.Containers[0].Image)
+							t.Logf("TimeTaken:%s", imagePullTime)
```

Comment:
I wonder if this logging could be better than:
```
    common_suite_test.go:179: Pod with larger image started successfully
    common_suite_test.go:180: ImageName:quay.io/sudharshanibm3/test-images:largeimage
    common_suite_test.go:181: TimeTaken:25s
```
Maybe it could go into a single log line and be more like Time taken to pull 4GB image: ?